app/controllers/employment/EmployerInformationController.scala

```diff
@@ -18,17 +18,65 @@ package controllers.employment
 
 import actions.AuthorisedAction
 import config.AppConfig
+import controllers.employment.routes._
+import controllers.studentLoans.routes._
+import controllers.taxableLumpSums.routes._
 import models.employment.EmploymentSourceOrigin
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import play.mvc.Call
 import services.EmploymentSessionService
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import utils.{InYearUtil, SessionHelper}
 import views.html.employment.EmployerInformationView
 
 import javax.inject.Inject
+import scala.Option.{unless, when}
 import scala.concurrent.Future
 
+case class EmployerInformationRow(
```

Comment:
Is this the right place to put this? Feels like some view structures that I'd have in a separate place? Even if this is the place for them to be created